lib/Pusher.php

```diff
@@ -20,9 +20,9 @@
 		+ Phil Leggetter (phil@leggetter.co.uk)
 */
 
-class PusherException extends Exception
-{
-}
+require_once('PusherException.php');
+require_once('PusherHTTPException.php');
+require_once('TriggerResult.php');
```

Comment:
Autoloading does not require namespacing. these are 2 independant concerns (for instance, Twig relies on autoloading since at least 4 years without using namespacing).